tools/quic/certs/leaf_cert.pem

```diff
@@ -0,0 +1,83 @@
+Certificate:
```

Comment:
Would it be possible to use the WPT CA to generate this key pair? https://github.com/web-platform-tests/wpt/tree/master/tools/certs Other browsers may rely on having the CA installed instead of using a flag to ignore cert errors.

And could you write some instructions on how to generate/update them? Also it seems that this one has already expired?

Reply:
Done, I think. Now we have cert.pem and cert.key.

Reply:
Hmm, The server doesn't work with the new certificate. I'll continue debugging tomorrow, I'll appreciate any insights. @vasilvv

I created the old files with https://www.chromium.org/quic/playing-with-quic.

Reply:
aiortc/aioquic#89